docs/geedocs/5.3.3/answer/releaseNotes/relNotesGEE5_3_3.html

```diff
@@ -341,7 +419,7 @@ <h1>Release notes: Open GEE 5.3.3<a class="headerlink" href="#release-notes-open
 <td>Before uninstalling geserver verify if it’s running or not.</td>
 </tr>
 <tr class="row-even"><td>590</td>
-<td>Maps API Javascript Files Not Found</td>
+<td>Maps API JaVaScRiPt Files Not Found</td>
```

Comment:
Is that the correct syntax ?

Reply:
Well, I noticed Kerry approved...hmm...

Reply:
I think this is a small bit of humor. I don't see any harm